hermes-frontend/src/main/java/pl/allegro/tech/hermes/frontend/buffer/BackupMessagesLoader.java

```diff
@@ -230,15 +207,4 @@ public void onPublished(Message message, Topic topic) {
             }
         });
     }
-
-    private Optional<CachedTopic> loadCachedTopic(String topicQualifiedName, Executor executor) {
```

Comment:
I simplified this as `topicsCache.getTopic()` returns already loaded data (with `start()` upon cached object creation) and there is no need for an async operation.